java/code/src/com/redhat/rhn/frontend/security/PxtAuthenticationService.java

```diff
@@ -48,6 +47,7 @@
         set.add("/rhn/Login");
         set.add("/rhn/ReLogin");
```

Comment:
Those two we URLs were removed from `struts-config.xml` below, so it might be safe to remove them from this set as well?

Reply:
We removed them from struts but we enabled via Apache Rewrite a redirect of those URLs to the new login page, just for safekeeping.

Reply:
Okay, but why would that new redirect be needed exactly? If someone has a bookmark to those URLs they should update it, it's okay from my perspective that an old bookmark stops working. If there is broken links from inside the application those should be fixed, and with a redirect in place we might never find out if there is a broken link somewhere. Am I missing some other usecase?

Reply:
OK, removed ReLogin and made it point to the new login page.
Login2 must stay instead: it is used for IPA authentication: the first login fails (expected, it checks against Kerberos) and we require another authentication (using local authentication).